arrays/array_practice.rb

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env ruby
+
+myarray = [1,2,3,4,5,6,7,8,9,10]
+myarray.each do |number|
+  print number.to_s + "..."
+end
+print"\n"
+print "  T-"
+myarray.reverse.each do |number|
+  if number > 1
+  print number.to_s + ", "
+  end
+end
+puts "1...  BLASTOFF! \n"
+puts "The last element is " + myarray.last.to_s
+puts "The first element is " + myarray.first.to_s
+puts "The third element is " + myarray[2].to_s
+puts "The element with an index of 3 is " + myarray[3].to_s
+puts "The second from last element is " + myarray[8].to_s
```

Comment:
Can you figure out a way to do it without depending on the array being exactly 10 items long?  i.e. is there a way to index from the end of the array?